BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -190,3 +192,44 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
     }
     
 }
+
+extension StoresTableViewController {
+    
+    func setupRefreshControl() {
+        
+        refreshControl = UIRefreshControl()
+        refreshLoadingView = UIView(frame: self.refreshControl!.bounds)
+        
+        refreshLoadingView.backgroundColor = UIColor.clear
```

Comment:
Don't really see you using this `refreshLoadingView` anywhere. You could probably do without it.